recipes/multiping/meta.yaml

```diff
@@ -3,19 +3,19 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
```

Comment:
Please remove the comments in the recipe.

Reply:
Done.